### DISCUSSION TITLE
Can't locate developer settings anymore in settings...

### DISCUSSION THREAD

**Discussion by MKindig97** [phantom/docs#167] [Help & Support] [answered]
### Summary

There used to be a drop down option to access Developer Settings such as changing your networks to test-mode. I see this piece of functionality is no longer available. I don't know what to do as I'm following a tutorial on how to build a CashApp clone. Any input would be greatly appreciated, thanks!

### Steps to Reproduce

![Screenshot (141)](https://github.com/phantom/docs/assets/126621157/c7866d8a-3cd8-4f4d-a8fa-46f1e5ee0583)

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
@MKindig97 Would you mind attempting to scroll down. There should be a few more settings option if you scroll down in the extension.

**Reply by nik4nik** [reply]
I had this problem with a touchpad without a mouse.
It is really impossible to scroll down.
Ways to solve it:
1. Set up the touchpad to be able to scroll down.
2. Connect a mouse with a wheel.
3. Install another wallet. For example, Solflare does not have this problem.
